cmake/Platform/Common/Install_common.cmake

```
@@ -492,6 +492,15 @@ function(ly_setup_cmake_install)
         list(JOIN relative_templates ",\n${indent}" O3DE_INSTALL_TEMPLATES)
     endif()
 
+    # Read the "repos" key from the source engine.json
+    o3de_read_json_array(engine_repos ${LY_ROOT_FOLDER}/engine.json "repos")
+    if(engine_repos)
+        foreach(repo_uri ${engine_repos})
+            list(APPEND repos "\"${repo_uri}\"")
+        endforeach()
+        list(JOIN repos ",\n${indent}" O3DE_INSTALL_REPOS)
```

Can you also sort the repo URIs as well?
```suggestion
        list(SORT repos CASE INSENSITIVE)
        list(JOIN repos ",\n${indent}" O3DE_INSTALL_REPOS)
```